google/cloud/spanner_dbapi/cursor.py

```diff
@@ -257,22 +273,25 @@ def _raise_if_closed(self):
             raise InterfaceError("cursor is already closed")
 
     def close(self):
-        """Close this cursor.
+        """Close cursor of linked Database.
```

Comment:
"Close this cursor" sounds better. There can be a lot of cursors to a single database